### ISSUE TITLE
Issue with protobuf: _pb2.py file, generated code is out of date

### ISSUE THREAD

**Issue by GilShoshan94** [FirebirdSQL/python3-base#1]
Hi @pcisar,

I freshly installed firebird 4 and and a new conda environment with python 3.10.
I pip installed firebird-driver and firebird-lib.

While following the [Getting Started documentation](https://firebird-driver.readthedocs.io/en/latest/getting-started.html#connecting-to-a-database)

I ran into an error message at the first line of code:

```
(django)PS C:\WINDOWS\system32> python
Python 3.10.4 | packaged by conda-forge | (main, Mar 24 2022, 17:32:50) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from firebird.driver import connect
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\PC-Gil\Miniconda3\envs\django\lib\site-packages\firebird\driver\__init__.py", line 38, in <module>
    from .config import driver_config, ServerConfig, DatabaseConfig, DriverConfig
  File "C:\Users\PC-Gil\Miniconda3\envs\django\lib\site-packages\firebird\driver\config.py", line 43, in <module>
    from firebird.base.config import Config, StrOption, IntOption, BoolOption, EnumOption, \
  File "C:\Users\PC-Gil\Miniconda3\envs\django\lib\site-packages\firebird\base\config.py", line 61, in <module>
    from .config_pb2 import ConfigProto
  File "C:\Users\PC-Gil\Miniconda3\envs\django\lib\site-packages\firebird\base\config_pb2.py", line 36, in <module>
    _descriptor.FieldDescriptor(
  File "C:\Users\PC-Gil\Miniconda3\envs\django\lib\site-packages\google\protobuf\descriptor.py", line 560, in __new__
    _message.Message._CheckCalledFromGeneratedFile()
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).

More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates
>>>
```

I have no experience with protobuf but this is confusing for me.

Did I do anything wrong ?

My python environment
---
```
(django)PS C:\WINDOWS\system32> conda list
# packages in environment at C:\Users\PC-Gil\Miniconda3\envs\django:
#
# Name                    Version                   Build  Channel
bzip2                     1.0.8                h8ffe710_4    conda-forge
ca-certificates           2022.5.18.1          h5b45459_0    conda-forge
firebird-base             1.3.1                    pypi_0    pypi
firebird-driver           1.5.0                    pypi_0    pypi
firebird-lib              1.2.0                    pypi_0    pypi
libffi                    3.4.2                h8ffe710_5    conda-forge
libzlib                   1.2.12               h8ffe710_0    conda-forge
openssl                   3.0.3                h8ffe710_0    conda-forge
pip                       22.1.2             pyhd8ed1ab_0    conda-forge
protobuf                  4.21.1                   pypi_0    pypi
python                    3.10.4          hcf16a7b_0_cpython    conda-forge
python-dateutil           2.8.2                    pypi_0    pypi
python_abi                3.10                    2_cp310    conda-forge
setuptools                62.3.4          py310h5588dad_0    conda-forge
six                       1.16.0                   pypi_0    pypi
sqlite                    3.38.5               h8ffe710_0    conda-forge
tk                        8.6.12               h8ffe710_0    conda-forge
tzdata                    2022a                h191b570_0    conda-forge
ucrt                      10.0.20348.0         h57928b3_0    conda-forge
vc                        14.2                 hb210afc_6    conda-forge
vs2015_runtime            14.29.30037          h902a5da_6    conda-forge
wheel                     0.37.1             pyhd8ed1ab_0    conda-forge
xz                        5.2.5                h62dcd97_1    conda-forge
(django)PS C:\WINDOWS\system32>
```

**Comment by pcisar** [comment]
Hi, the firebird-base pqackage uses protobuf 19.0.x. Unfortunately, python code generated by protobuf 20.0.x is not backward compatible. As firebird-base protobuf dependency in setup.cfg does not have upper bound, pip installs the latest one (20.x). Downgrade to protobuf 19.0 or upgrade to just released firebird-base 1.4.0

**Comment by GilShoshan94** [comment]
Thank you.
